src/vscode-dts/vscode.proposed.chatParticipantAdditions.d.ts

```diff
@@ -245,6 +245,23 @@ declare module 'vscode' {
 		 * The chat extension should not activate if it doesn't support the current version.
 		 */
 		export const _version: 1 | number;
+
+		export function registerChatParticipantDetectionProvider(participantDetectionProvider: ChatParticipantDetectionProvider): Disposable;
```

Comment:
Another idea I just thought of is that this would make sense on the default participant, like `welcomeMessageProvider` etc, but I think this shape is fine too.